From a735b5c09de7b6925267d2ff6514a9ba06be921d Mon Sep 17 00:00:00 2001
From: Jyoti Mahapatra <jyotima@amazon.com>
Date: Tue, 9 Mar 2021 05:55:57 +0000
Subject: --EKS-PATCH-- Allow override of kube-proxy base image

Signed-off-by: Jyoti Mahapatra <jyotima@amazon.com>
---
 build/common.sh | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/build/common.sh b/build/common.sh
index f3cb40448c8..4bda958b0a4 100755
--- a/build/common.sh
+++ b/build/common.sh
@@ -107,12 +107,13 @@ readonly KUBE_BUILD_SETCAP_IMAGE="${KUBE_BUILD_SETCAP_IMAGE:-$KUBE_BASE_IMAGE_RE
 #
 # $1 - server architecture
 kube::build::get_docker_wrapped_binaries() {
+  local go_runner_version=buster-v2.3.1
   ### If you change any of these lists, please also update DOCKERIZED_BINARIES
   ### in build/BUILD. And kube::golang::server_image_targets
   local targets=(
-    "kube-apiserver,${KUBE_APISERVER_BASE_IMAGE}"
-    "kube-controller-manager,${KUBE_CONTROLLER_MANAGER_BASE_IMAGE}"
-    "kube-scheduler,${KUBE_SCHEDULER_BASE_IMAGE}"
+    "kube-apiserver,${KUBE_BASE_IMAGE_REGISTRY}/go-runner:${go_runner_version}"
+    "kube-controller-manager,${KUBE_BASE_IMAGE_REGISTRY}/go-runner:${go_runner_version}"
+    "kube-scheduler,${KUBE_BASE_IMAGE_REGISTRY}/go-runner:${go_runner_version}"
     "kube-proxy,${KUBE_PROXY_BASE_IMAGE}"
   )

--
2.16.6

