From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Vineeth Bandi <banvinee@amazon.com>
Date: Tue, 14 Jan 2025 11:52:26 -0600
Subject: [PATCH] skip x509 tests

Signed-off-by: Vineeth Bandi <banvinee@amazon.com>
---
 hack/make-rules/test.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hack/make-rules/test.sh b/hack/make-rules/test.sh
index e9074678a8f..c78fc9b3b60 100755
--- a/hack/make-rules/test.sh
+++ b/hack/make-rules/test.sh
@@ -54,7 +54,7 @@ kube::test::find_dirs() {
         \) -prune \
       \) -name '*_test.go' -print0 | xargs -0n1 dirname | sed "s|^\./|${KUBE_GO_PACKAGE}/|" | LC_ALL=C sort -u

-    find ./staging -name '*_test.go' -not -path '*/test/integration/*' -prune -print0 | xargs -0n1 dirname | sed 's|^\./staging/src/|./vendor/|' | LC_ALL=C sort -u
+    find ./staging -name '*_test.go' -not -path '*/test/integration/*' -not -path '*/apiserver/pkg/authentication/request/x509/*' -prune -print0 | xargs -0n1 dirname | sed 's|^\./staging/src/|./vendor/|' | LC_ALL=C sort -u
   )
 }

-- 
2.46.0

